share/spice/yoga_asanas/yoga_asanas.css

```diff
@@ -0,0 +1,15 @@
+.zci--yoga_asanas img {
+	max-width: none;
+}
+
+.zci--yoga_asanas .tile__media__img {
+	height: 125%;
+	position: absolute;
+	bottom: -10px;
+	left: 50%;
+	transform: translate(-50%,0);
+}
+
+.zci--yoga_asanas .tile__media {
+	height: 14em;
```

Comment:
I think we may need to reduce this for mobile devices? In landscape, the initial view of the page has the tiles clipped:

![2015-08-26 18 55 01](https://cloud.githubusercontent.com/assets/873785/9509156/540c1f0a-4c2a-11e5-86a6-b9b7f43bbc22.png)